eventually/.pylintrc

```diff
@@ -330,7 +330,7 @@ ignore-comments=yes
 ignore-docstrings=yes
 
 # Ignore imports when computing similarities.
-ignore-imports=no
+ignore-imports=yes
```

Comment:
???

Reply:
i don`t know how to resolve duplicate code problem